include/enet/types.h

```diff
@@ -5,9 +5,11 @@
 #ifndef __ENET_TYPES_H__
 #define __ENET_TYPES_H__
 
+
 typedef unsigned char enet_uint8;       /**< unsigned 8-bit type  */
 typedef unsigned short enet_uint16;     /**< unsigned 16-bit type */
 typedef unsigned int enet_uint32;      /**< unsigned 32-bit type */
+typedef u_int64_t enet_uint64;      /**< unsigned 64-bit type */
```

Comment:
`u_int64_t` isn't portable: https://stackoverflow.com/q/42979055/2038264
Should use `uint64_t` (and also include stdint.h)